modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/resources/META-INF/resources/js/data_set/settings/Settings.tsx

```diff
@@ -119,9 +122,30 @@ const Settings = ({
 		setLoading(false);
 	};
 
+	const getShowManagementBarInEmptyState = async () => {
+		const url = getDataSetResourceURL({
+			dataSetERC: dataSet.externalReferenceCode,
+			params: {
+				fields: 'showManagementBarInEmptyState',
+			},
+		});
+
+		const response = await fetch(url, {
+			headers: DEFAULT_FETCH_HEADERS,
```

Comment:
We'll need to join this with the fetch we make above, to use a single call to retrieve the data set object entry.

This was not needed in the past because this had only one setting